service/src/test/java/uk/gov/hmcts/reform/fpl/service/PopulateCaseServiceTest.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.jackson.JacksonAutoConfiguration;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.fpl.model.common.DocumentReference;
+import uk.gov.hmcts.reform.fpl.service.time.Time;
+import uk.gov.hmcts.reform.fpl.utils.FixedTimeConfiguration;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = {FixedTimeConfiguration.class, JacksonAutoConfiguration.class,
+    PopulateCaseService.class})
+class PopulateCaseServiceTest {
```

Comment:
personally i think that test for test util class is not necessary needed
It can be excluded from code coverage

Reply:
If it's already there I'd leave it. Also, when some changes are made accidentally we'll see it failing on pipeline quite quickly rather than on E2E step.